sql/firefox_desktop_visited_5_uri_mau28_by_dimensions_v1.sql

```diff
@@ -0,0 +1,52 @@
+WITH
```

Comment:
Is this broken out as a separate table simply to keep it separate from the current clients_last_seen pipeline? Or do you see a need to maintain separate tables per usage criterion?

Reply:
to keep it separate from the current clients_last_seen pipeline

Reply:
We should probably call that out explicitly in a comment at the top of this file.